### ISSUE TITLE
Benchmarks started to error out

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#7149]
https://github.com/datalad/datalad/actions/workflows/benchmarks.yml
seems first to trigger during https://github.com/datalad/datalad/pull/7147 but those changes do not relate, probably due to some other changes, may be in -deprecated

```
· Discovering benchmarks
·· Error running /opt/hostedtoolcache/Python/3.7.15/x64/bin/python /opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/asv/benchmark.py discover /home/runner/work/datalad/datalad/benchmarks /tmp/tmp43rsjm9j/result.json (exit status 1)
   STDOUT -------->
   
   STDERR -------->
   Traceback (most recent call last):
     File "/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/asv/benchmark.py", line 1435, in <module>
       main()
     File "/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/asv/benchmark.py", line 1428, in main
       commands[mode](args)
     File "/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/asv/benchmark.py", line [110](https://github.com/adswa/datalad/actions/runs/3411262182/jobs/5675245810#step:7:111)3, in main_discover
       list_benchmarks(benchmark_dir, fp)
     File "/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/asv/benchmark.py", line 1088, in list_benchmarks
       for benchmark in disc_benchmarks(root):
     File "/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/asv/benchmark.py", line 996, in disc_benchmarks
       class_attr)
     File "/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/asv/benchmark.py", line 931, in _get_benchmark
       instance = klass()
     File "/home/runner/work/datalad/datalad/benchmarks/common.py", line 74, in __init__
       from datalad.interface import ls
   ImportError: cannot import name 'ls' from 'datalad.interface' (/home/runner/work/datalad/datalad/datalad/interface/__init__.py)
```

can't look ATM myself.

**Comment by bpoldrack** [comment]
> ImportError: cannot import name 'ls' from 'datalad.interface'

`ls` moved to deprecated. Only question is why it's failing only now (or may be we didn't notice)

**Comment by bpoldrack** [comment]
Fixing the import in `maint`, but I think those benchmarks should be deleted in core (`master`). This package doesn't have `ls` and there's no intention to develop it (that's why it's in deprecated). Don't see a point in the benchmarks relying on datalad-deprecated really. If we want that, those benchmarks should run in deprecated instead. Agree, @yarikoptic?

**Comment by yarikoptic** [comment]
note that one use of `ls` could be entirely dropped in the `__init__` for older datalad versions...
Ideally should be "moved" not "deleted", so along with benchmarking setup etc... but -- whatever, I will not fight against removal.

**Comment by bpoldrack** [comment]
@yarikoptic 

Just FTR: With "remove" I only referred to datalad core. Whether to set this up in deprecated is independent. I am obviously fine with "move", but I personally won't bother putting work into deprecated. Removal here does not prohibit introduction in deprecated.

**Comment by yarikoptic-gitmate** [comment]
Issue fixed in [`0.17.10`](https://github.com/datalad/datalad/releases/tag/0.17.10)
